api/v1alpha1/helmchartproxy_webhook.go

```diff
@@ -45,6 +50,16 @@ func (p *HelmChartProxy) Default() {
 	if p.Spec.ReleaseNamespace == "" {
 		p.Spec.ReleaseNamespace = "default"
 	}
+
+	// If 'Wait' is set, we need to set default 'Timeout' to make install successful.
+	if p.Spec.Options != nil && p.Spec.Options.Wait {
+		p.Spec.Options.Timeout = &metaV1.Duration{Duration: time.Second * 600}
```

Comment:
Should we make this timeout a user-configurable flag so that when they enable wait they can also configure the upper time limit?

Reply:
Yes. My bad, forgot to add `p.Spec.Options.Timeout  == nil` check.

Reply:
Updated. Please review.

Reply:
let's make this default timeout value a const so that we can easily write a UT and validate the default flow from a single source of truth